package.json

```
@@ -1,22 +1,31 @@
 {
   "name": "@tduniec/backstage-timesaver-plugin",
-  "version": "1.0.0",
+  "version": "1.0.1",
+  "description": "This plugin provides an implementation of charts and statistics related to your time savings that are coming from usage of your templates.",
+  "private": true,
   "engines": {
     "node": "18 || 20"
   },
   "workspaces": {
-  "packages": [
-    "plugins/*"
-  ]},
+    "packages": [
+      "plugins/*"
+    ]
+  },
   "repository": "https://github.com/tduniec/backstage-timesaver-plugin.git",
   "author": "tduniec <tomekduniec@gmail.com>",
   "license": "Apache-2.0",
   "scripts": {
     "build": "yarn workspaces run build",
     "clean": "yarn workspaces run clean",
+    "tsc": "tsc",
     "test": "yarn workspaces run test",
-    "lint": "eslint .",
-    "tsc": "tsc"
+    "test:all": "backstage-cli repo test --coverage",
+    "fix": "backstage-cli repo fix",
+    "lint": "backstage-cli repo lint --since origin/main",
```

@tduniec, this is also related to my comment above. I believe it would be better to use backstage-cli linting configurations to better comply with Backstage standards. Moreover, Backstage linting configurations also look for copyright headers and other stuff using their scripts folder when doing so, which eslint won't do by itself.
```suggestion
    "lint": "backstage-cli repo lint --since origin/main",
```

---

that makes sense, I will adjust soon